public_site/src/site/markdown/index.md

```diff
@@ -19,6 +19,10 @@ wcm.io DevOps is an Open Source project which provides tools for configuration m
 * [Ansible Automation for AEM](ansible-aem/)
 * [Jenkins Pipeline Library for CI/CD](https://github.com/wcm-io-devops/jenkins-pipeline-library)
 
+##### Cloud Manager
```

Comment:
we might but it directly unter "CI/CD and IT Automation" instead of creating a new catgory?

Reply:
Sure.